netboot/main.go

```diff
@@ -33,6 +37,8 @@ var (
 	readTimeout        = flag.Int("timeout", 3, "Read timeout in seconds")
 	dhcpRetries        = flag.Int("retries", 3, "Number of times a DHCP request is retried")
 	userClass          = flag.String("userclass", "", "Override DHCP User Class option")
+	caCertFile         = flag.String("cacerts", "/sys/firmware/vpd/ro/ca_certs", "Base64 encoded CA cert file")
```

Comment:
I think we should not set a default here, because VPD is just one of the possible backends (NVRAM backend coming soon). Other LinuxBoot users are not using VPD at all. I'd rather force the user to specify the path, which for simplicity can be wrapped in uinit (the only component that assumes the presence of VPD)

Reply:
Of course, sounds good.